### DISCUSSION TITLE
Collabra config not working

### DISCUSSION THREAD

**Discussion by MarcS1975** [nextcloud/all-in-one#4012] [Questions] [answered]
### Steps to reproduce
1. Installed Collabra container
2. I am using cloudflare as my doimain host but i am strugling to find  **_settings/admin/richdocuments_** to update my wopi-allowlist

### Expected behavior 
Collabra available in NC AIO
### Actual behavior 
Collabra container Log file :

`wsd-00007-00007 2024-01-08 00:21:22.863623 +0000 [ coolwsd ] TRC  Have 1 new children.| wsd/COOLWSD.cpp:5919
stderr
08/01/2024
00:21:22
wsd-00007-00007 2024-01-08 00:21:22.863643 +0000 [ coolwsd ] INF  WSD initialization complete: setting log-level to [warning] as configured.| wsd/COOLWSD.cpp:5935
stderr
08/01/2024
00:21:22
wsd-00007-00007 2024-01-08 00:21:22.864070 +0000 [ coolwsd ] WRN  Waking up dead poll thread [main], started: false, finished: false| net/Socket.hpp:714
stderr
08/01/2024
00:21:22
wsd-00007-00014 2024-01-08 00:21:22.891480 +0000 [ remotefontconfig_poll ] ERR  Remote config server has response status code: 502 (Bad Gateway)| wsd/COOLWSD.cpp:1306
stderr
08/01/2024
00:22:10
wsd-00007-00016 2024-01-08 00:22:10.665362 +0000 [ websrv_poll ] ERR  Poco::Net::DNS::resolve("xxxx") failed: Host not found: xxxx| wsd/COOLWSD.cpp:4004
stderr
08/01/2024
00:22:10
wsd-00007-00016 2024-01-08 00:22:10.665390 +0000 [ websrv_poll ] WRN  convert-to: Requesting address is denied: xxx| wsd/COOLWSD.cpp:4010
`

### Host OS 
Debian

#### Nextcloud AIO version 
27
#### Other valuable info 
The documentation says
`add all [Cloudflare IP-ranges](https://www.cloudflare.com/ips/) to the wopi-allowlist in https://yourdomain.com/settings/admin/richdocuments`

but I do not know where I can find **_settings/admin/richdocuments_**

**Comment by MarcS1975** [comment, marked as answer]
Found the solution:
The wopi-allowlist can be accessed within Nextcloud via **Administration Settings - Office - Advanced Settings
_Allow list for WOPI requests_**
You can add the [Cloudflare IP ranges ](https://www.cloudflare.com/ips-v4/#)there, update the settings and restart the Collabra container.

**Reply by zEdS15B3GCwq** [reply]
I followed these instructions (i.e. the ones on the reverse proxy readme), added all the IP masks from that list and any I could think of, but WOPI still doesn't work (vs empty WOPI or something like 255.0.0.0/1 does, so the issue is with the addresses). Is there a way in AIO to debug which IP is actually failing? I guess Collabora logs could be set to a more verbose level, but that's not straightforward within AIO.

Is it really necessary to restart the container? Changes in the UI seemed to have an immediate effect even without that.

Edit: right after I wrote this, I found this discussion about debugging code: https://github.com/nextcloud/all-in-one/discussions/1358 . It seems the answer I'm looking for is there.
